cardano-cli/src/Cardano/CLI/EraBased/Run/Query.hs

```diff
@@ -826,37 +833,48 @@ writeStakeAddressInfo
   :: Maybe (File () Out)
   -> DelegationsAndRewards
   -> Map StakeAddress Lovelace -- ^ deposits
+  -> Map StakeAddress (L.DRep L.StandardCrypto) -- ^ vote delegatees
   -> ExceptT QueryCmdError IO ()
 writeStakeAddressInfo
   mOutFile
   (DelegationsAndRewards (stakeAccountBalances, stakePools))
-  stakeDelegDeposits =
+  stakeDelegDeposits
+  voteDelegatees =
     firstExceptT QueryCmdWriteFileError . newExceptT
       $ writeLazyByteStringOutput mOutFile (encodePretty jsonInfo)
  where
   jsonInfo :: [Aeson.Value]
   jsonInfo =
     map
-      (\(addr, mBalance, mPoolId, mDeposit) ->
+      (\(addr, mBalance, mPoolId, mDRep, mDeposit) ->
         Aeson.object
         [ "address" .= addr
-        , "delegation" .= mPoolId
+        , "stakeDelegation" .= mPoolId
+        , "voteDelegation" .= fmap friendlyDRep mDRep
```

Comment:
can you add a golden test for this command output please?

Reply:
Ah right. I don't know why I thought this was an offline command. This needs to be a part of a separate change to`cardano-testnet`.